.github/kokoro/ice40.sh

```diff
@@ -11,7 +11,9 @@ echo "Running ice40 tests"
 echo "----------------------------------------"
 (
 	cd build
-	VPR_NUM_WORKERS=${CORES} make -j ${MAX_CORES} --output-sync=target \
-		all_ice40
+	export VPR_NUM_WORKERS=${CORES}
```

Comment:
This doesn't belong in this PR

Reply:
Its in a separate pull request at https://github.com/SymbiFlow/symbiflow-arch-defs/pull/785 that this is based on.